src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.13.0.2733")]
```

Comment:
Please revert the changes to this file.

Reply:
Good point! @reddyashish since I happen to be reading this, addressed and made this PR ready for review